resources/views/users/show-introduction.blade.php

```diff
@@ -12,9 +12,25 @@
     </h2>
 
     <!-- Video -->
-    <div class="frame">
-        {{ $user->introduction() }}
-    </div>
+    @if (array_key_exists(locale(), $user->introduction()))
+        <div class="frame">
+            <div class="stack w-full" x-data="vimeoPlayer({
+                url: '{{ $user->introduction()[locale()] }}',
+                byline: false,
+                dnt: true,
+                pip: true,
+                portrait: false,
+                responsive: true,
+                speed: true,
+                title: false
+            })" @ended="player().setCurrentTime(0)">
+            </div>
+        </div>
+    @else
+        <div class="frame">
+            {{ __('Introduction video') }}
+        </div>
+    @endif
```

Comment:
I wonder if we should fall back to a default language if a locale doesn't exist. Maybe the English one? Which is what I think happens if the site isn't localized in a given language. I suppose if the `locale()` and `en` don't exist, then we don't include anything, including the `h2` that says "Introduction video` and just leave the site to say welcome and the Continue button at the bottom. 

Does that seam reasonable? Let me know what you think, or if I'm missing anything.

Reply:
Thanks @jobara, your suggestion sounds good to me and would simplify code blocks. I will update it like your suggestion.